includes/Core/Util/Migration_1_45_0.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * Migration for 1.45.0
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Authentication\Authentication;
+use Google\Site_Kit\Core\Authentication\Google_Proxy;
+use Google\Site_Kit\Core\Authentication\Profile;
+use Google\Site_Kit\Core\Authentication\Verification_File;
+use Google\Site_Kit\Core\Authentication\Verification_Meta;
+use Google\Site_Kit\Core\Permissions\Permissions;
+use Google\Site_Kit\Core\Storage\Options;
+use Google\Site_Kit\Core\Storage\User_Options;
+use WP_User;
+use WP_Error;
+
+/**
+ * Class Migration_1_45_0
+ *
+ * @since 1.45.0
+ * @access private
+ * @ignore
+ */
+class Migration_1_45_0 {
+	/**
+	 * Target DB version.
+	 */
+	const DB_VERSION = '1.45.0';
+
+	const OPTION_KEY_PSI_UPDATED = 'googlesitekit_psi_updated';
```

Comment:
> A potential way to solve this is could be:
> * Include an upgrade routine that sets a very simple option in the DB like googlesitekit_psi_updated if the site is currently connected.
> * **Show the tooltip tour only if that flag is set (and if PSI is not in the active modules option).**
> * On reset, delete this option so that it's gone.

We also need to check this flag when we determine a value for the `psiManuallyActivated` property.

Reply:
https://github.com/google/site-kit-wp/pull/4305/commits/43021fd8f08fdf722074790b62021424c4a7db18